tools/monorepo-utils/src/ci-jobs/lib/config.ts

```diff
@@ -251,11 +256,21 @@ export interface TestJobConfig extends BaseJobConfig {
 	 */
 	type: JobType.Test;
 
+	/**
+	 * The type of the test.
+	 */
+	testType: string;
```

Comment:
```suggestion
	testType: typeof testTypes[number];
```

Paired with the const assertion on `testTypes`, this constrains `testType` to one of the values within the `testTypes` array.